luda-editor/luda-editor-client/src/app/editor/clip_editor/camera_clip_editor/wysiwyg_editor/character_wysiwyg_editor/mod.rs

```diff
@@ -8,7 +8,11 @@ use crate::app::types::*;
 use namui::prelude::*;
 use std::sync::Arc;
 
-pub struct CharacterWysiwygEditor {}
+pub struct CharacterWysiwygEditor {
+    id: String,
+    resizer: Resizer,
+    cropper: Cropper,
+}
```

Comment:
Are these changes intended to be included in this pr?

Reply:
Yes, to solve the warning in luda-editor-client. code itself has no meaning.